### ISSUE TITLE
Can you open a branch for me?

### ISSUE THREAD

**Issue by cool-little-fish** [gyhua96/Sniffer#1]
I hava added a function which can write all the data package into Disk to record your action.
So if I could get one branch from you in this project?If you see this message,please leave a message 
to me by email to spirit.swc@gmail.com  Thanks a lot.

**Comment by gyhua96** [comment]
Sorry being busy these days. I don't use branches a lot. I created a branch named "dev", is that what you want? Thanks for visiting my project!

**Comment by cool-little-fish** [comment]
That's alright.And Actually I didn't use branch before,but I want to have a try.So I put a request in the issues.

Last two weeks I was busy with school tests.you kown it's really ....how to say...but I have to quote 'We do what we have to do,then I could do what we want to do',Although I did not like these courses very much,I still want to learn something from it with a good attitude what I think it's very import for life.

And I read all your codes about this project.Thanks for showing your codes.I added some codes based in your codes aiming to write all the packages into a text file.It works fine in FX 2.x version.But when I upgrade my FX version to 3.0.It will fail to run.I did not solve this problem yet..

Thanks for opening a branche.Now I am going to put  my codes to your pull request.It's up to you if to 
merge any branches codes to your master branchs.
